app/tools/android/make_apk.py

```diff
@@ -757,6 +776,13 @@ def main(argv):
           'Mac OS respectively. For example, '
           '--extensions=/path/to/extension1:/path/to/extension2.')
   group.add_option('--extensions', help=info)
+  info = ('The native external extension path. Put the architecture-specific '
+          'library file in the corresponding folder (armeabi-v7a/x86) under '
```

Comment:
There are more archs: x86-64, arm64 to be supported. I think you need verify whether the .so arch is matched with current app_template.